docs/data/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -43,102 +55,87 @@ apiRef.current.subscribeEvent(
 );
 
 // Hook subscription (only available inside the scope of the grid)
-useGridApiEventHandler('${event.name}', onEvent);    
+useGridApiEventHandler('${event.name}', onEvent);
+${propExample}
 `;
   }, [event]);
 
-  const planBadge = React.useMemo(() => {
-    if (event.projects.includes('x-data-grid')) {
+  return <HighlightedCode code={example} language="tsx" />;
+};
+
+const COLUMNS = [
+  {
+    field: 'name',
+    headerName: 'Name',
+    width: 240,
```

Comment:
I did not do that because it takes more space and on smaller screen the size for the description was starting to be small.
But we could put a `code` and make the size smaller.

Reply:
I moved the description to the `body2` variant and it seems to be enough for now.